src/OV/System.hpp

```diff
@@ -3,12 +3,21 @@
 
 #pragma once
 
+class ProfileMap;
+class Path;
+
 enum class SSHStatus {
   ENABLED,
   DISABLED,
   TEMPORARY,
 };
 
+void
+LoadConfigFile(ProfileMap &map, Path path);
```

Comment:
Why use `ProfileMap` here? Is this a XCSoar profile? (There is no documentation explaining what this library is about)

Reply:
`Profile/Map.hpp` is a nice library that I wanted to use to read and write `config.uEnv` - mainly for updating the `rotation` variable. However, Its probably too much for this rather small task and I can just use `std::map` and a few of its member functions for the intended job.

Documentation will be added.